variables.tf

```diff
@@ -33,6 +33,16 @@ variable "dependency_ids" {
   default = {}
 }
 
+variable "app_autosync" {
+  type    = bool
+  default = true
+}
```

Comment:
Same comment I made here: https://github.com/camptocamp/devops-stack-module-traefik/pull/25#discussion_r1044200571

Reply:
After discussion, will leave as is for now. These changes are urgently needed for Scaleway.